common/src/main/res/values/strings.xml

```diff
@@ -1121,4 +1121,9 @@
     <string name="sensor_description_car_fuel_type">List of available fuel types for the connected car</string>
     <string name="basic_sensor_name_car_ev_connector_type">Car EV Connector Type</string>
     <string name="sensor_description_car_ev_connector_type">List of available EV connectors for the connected car</string>
+    <string name="aa_set_favorites">Select your favorite entities to appear in the Favorites category in the Android Auto/Automotive driving optimized app. You can also drag and drop to change the order in which they appear.</string>
```

Comment:
Should this include a note on that the number of favorites shown may be limit by the vehicle?

Still not a big fan of "Android Auto/Automotive driving optimized app". It is inconsistent with the `aa_favorites_summary` and I think "Home Assistant driving interface" would be simpler and good enough.

Reply:
updated and I agree thats simpler and good enough, thanks for the suggestion

Reply:
Update the aa_favorites_summary string as well to use "Home Assistant driving interface" instead of "driving optimized screen" for consistency?

Reply:
ah agreed, updated